Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -39,8 +39,38 @@
 #import "BPKCalendarStickyHeader.h"
 #import "BPKCalendarYearPill.h"
 
+
 NS_ASSUME_NONNULL_BEGIN
 
+// This is kept around only to support the React Native calendar implementation
```

Comment:
If we're removing the deprecated method, why not remove this for RN too and we can adopt the breaking change in `backpack-react-native`?

Reply:
Actually the current RN implementation actually relies on this API with no other option to migrate to imo